packages/terra-doc-template/src/ExampleTemplate.scss

```diff
@@ -0,0 +1,73 @@
+:local {
+  .template {
+    border: 1px solid 	#e8e8e8;
+    border-radius: 3px;
+    overflow: hidden;
+  }
+
+  .template + .template {
+    margin-top: 0.9375rem;
+  }
+
+  .header {
+    border-bottom: 1px solid #e8e8e8;
+    padding: 0.625rem;
+  }
+
+  .content {
+    padding: 0.625rem;
+  }
+
+  .description {
+    padding-bottom: 0.625rem;
+  }
+
+  .footer {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+  }
+
+  .code {
+    height: 0;
```

Comment:
I don't have exact transition values, but adding back an animation when toggling the src code would be cool.
```
transition: height <duration> <timing function>
```

Reply:
The animations in the original terra-toggle-button are done by react-animate-height. The only other way to animate the height is to set a generic max-height on the component. 

https://github.com/cerner/terra-core/blob/master/packages/terra-toggle/src/Toggle.jsx#L37

Because this component is becoming a hard dependency in every package I think we should remove as many dependencies as possible. If we want to experiment with a max-height solution there are a few caveats that come along with that.

Reply:
Ah gotcha, makes sense